internal/net/http/transport/option.go

```diff
@@ -31,12 +32,14 @@ var (
 	}
 )
 
+// WithRoundTripper returns the Option that set the RoundTripper when tr is not nil.
```

Comment:
Sorry I think this one is correct.
```suggestion
// WithRoundTripper returns the Option that set the RoundTripper.
```